drivers/util.py

```diff
@@ -1038,6 +1038,15 @@ def _isSCSIid(s):
     return regex.search(s, 0)
 
 
+def _is_usb_device(device):
```

Comment:
I think this function can be public, so without the first "_".

Reply:
Done.